src/high-level/lapack-templates.lisp

```diff
@@ -60,6 +60,17 @@
                 (storage target)
                 m)
                target)))))
+
+     (defmethod mult ((a ,vector-class) (b ,matrix-class) &key target (alpha ,(coerce 0 type)) (beta ,(coerce 1 type)) transa (transb :n))
+       (policy-cond:with-expectations (> speed safety)
+           ((type (member nil :n :t :c) transb)
+            (assertion (null transa)))
+           (let ((ta (ecase transb
+                       (:n :t)
+                       (:t :n)
+                       (:c (error "Specifying TRANSA to be :C is not supported for vector-matrix multiplication")))))
+             (mult b a :target target :alpha beta :beta alpha :transa ta :transb transa))))
```

Comment:
Is it typical of matrix libraries to do a left multiply this way? If `@` were to be implemented with right-associativity then it would end up being a mult of two vectors which might make more sense. Maybe there is some weirdness inherent in not storing whether a vector is a column or a row.

Reply:
Checking on matlab it doesn't seem to like `x*A*x`... What does numpy do?

Reply:
Numpy does left multiply this way:

```python
__main__> import numpy as np
__main__> x = np.array([0.5, 0.25, 0.25])
__main__> A = np.array([[1.0, 2.0, 3.0],
                        [1.0, 2.0, 3.0],
                        [1.0, 2.0, 3.0]])
__main__> np.dot(A,x)
array([1.75, 1.75, 1.75])
__main__> np.dot(x,A)
array([1., 2., 3.])
```

Reply:
(Etiquette question here: Am I the one that is meant to be clicking "resolved conversation"?)

Reply:
(If you fixed an issue, yes. If it's a question, no. :smile:)

Reply:
Gotcha, thanks. :)